### PR TITLE
roadmap page mobile responsive

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/RoadmapPage/RoadmapPage.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      label: "Curate collective",
+      date: "2023-02-28",
+    },
+    {
+      backImg: "/assets/Roadmap/pendants.svg",
+      label: "Airdrop pendants",
+      date: "2023-03-31",
+    },
+    {
+      backImg: "/assets/Roadmap/build.svg",
```
Comment (sweetmantech) [inline components/RoadmapPage/RoadmapPage.tsx]: optional: would be nice to split this out into it's own`.json` file to not clutter up the HTML.

love this architecture you've built. 🏗️ 

very easy to modify / update over time. ✏️

**Review by SweetmanTech — APPROVED** [review]
looks good to ship 🚢 

gonna review UX now

File: components/RoadmapPage/RoadmapPage.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import Slider from "../../shared/Slider"
+import Stage from "./Stage"
+import Layout from "../Layout"
+import { StageData } from "./types"
+import data from "./data.json"
+
+Swiper.use([Mousewheel])
+
+const RoadmapPage = () => {
+  let offset
```
Comment (sweetmantech) [inline components/RoadmapPage/RoadmapPage.tsx:13]: can you move this declaration inside your `useEffect` hook?

It's only being used there, correct?

File: components/RoadmapPage/data.json [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    "text": ""
+  },
+  {
+    "backImg": "/assets/Roadmap/redacted.svg",
+    "label": null,
+    "date": null,
+    "link": "/",
+    "text": "[REDACTED]"
+  }
+]
```
Comment (sweetmantech) [inline components/RoadmapPage/data.json:127]: 💯

File: components/RoadmapPage/data.json [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    "link": "/",
+    "text": "To get on the allowlist without a passport, take the Everything Corp Personality Quiz\nAssessment. Once your application is submitted, retweet your results to move up the\nreview process. Follow @cre8orsNFT to be notified once your application is accepted."
+  },
+  {
+    "backImg": "/assets/Roadmap/drop.svg",
+    "label": "drop trailer",
+    "date": "2023-07-07 UTC-4",
+    "certain": true,
+    "link": "/",
+    "text": ""
```
Comment (sweetmantech) [inline components/RoadmapPage/data.json]: missing `text` for the remaining boxes

**Review by SweetmanTech — APPROVED** [review]
small comments, overall looking good ✅

File: components/ClaimPage/ClaimPage.tsx [hunk 5]
```diff
@@ -1,46 +1,147 @@
...
+import Media from "../../shared/Media"
+import Footer from "../Footer"
+import { useTheme } from "../../providers/ThemeProvider"
+import useGradualFadeEffect from "../../hooks/useGradualFade"
+import Popover from "../../shared/Popover"
+import ConnectWallet from "./ConnetWallet"
+import Redeem from "./Redeem"
 
-function ClaimPage() {
+const ClaimPage = () => {
```
Comment (sweetmantech) [inline components/ClaimPage/ClaimPage.tsx]: same feedback as Claim Page

Why is Claim Page code in your pull request for Roadmap Page?

blocker: please remove Claim Page code from your Roadmap Page pull request 🚮

File: components/RoadmapPage/Stage.tsx [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+            rounded-[10px] md:rounded-[20px]
+            hover:bg-gradient-to-r hover:from-[#000000cf] hover:via-[#00000080] hover:to-[#000000cf]
+            bg-transparent
+            transition duration-[200ms]"
+          >
+            <div className="drop-shadow-[0_1.2px_1.2px_rgba(0,0,0,0.8)]">{stageData.text}</div>
+          </pre>
+        </div>
+      </a>
+    </div>
```
Comment (sweetmantech) [inline components/RoadmapPage/Stage.tsx:237]: this component for `<Stage />` is starting to get a little big & messy, would you agree?

Maybe we can make this more readable by making some reusable `child` components? 💭

Comment (sweetmantech) [reply components/RoadmapPage/Stage.tsx:237]: non-blocking, just a thought

File: pages/claim/index.tsx [hunk 7]
```diff
@@ -1,7 +1,5 @@
 import ClaimPage from "../../components/ClaimPage"
 
-function Claim() {
-  return <ClaimPage />
-}
+const Claim = () => <ClaimPage />
```
Comment (sweetmantech) [inline pages/claim/index.tsx]: please remove all files unrelated to `Roadmap Page` 🚮

**Review by SweetmanTech — CHANGES_REQUESTED** [review]
remove code unrelated to `Roadmap Page`
- remove Claim Page components 🚮

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**